timesketch/lib/datastores/elastic.py

```diff
@@ -88,7 +88,11 @@ def __init__(self, host='127.0.0.1', port=9200):
         self.ssl = current_app.config.get('ELASTIC_SSL', False)
         self.verify = current_app.config.get('ELASTIC_VERIFY_CERTS', True)
 
-        if self.ssl:
+        if self.ssl and not self.user and not self.password:
```

Comment:
To make it a little easier to read, maybe something like this?

```
if self.ssl:
    if self.user and self.password:
        self.client = Elasticsearch([{'host': host, 'port': port}],
                                        use_ssl=self.ssl,
                                        verify_certs=self.verify)
     else:
         self.client = Elasticsearch([{'host': host, 'port': port}],
                                        http_auth=(self.user, self.password),	                                     
                                        use_ssl=self.ssl

...

```

WDYT?

Reply:
Looks good. I'll update.